udata_croquemort/checker.py

```diff
@@ -24,13 +24,21 @@ def _format_response(self, response):
             check_date = response.get('updated')
             if check_date:
                 check_date = dateutil.parser.parse(response.get('updated'))
-            return {
+
+            result = {
                 'check:url': response.get('checked-url'),
                 'check:status': status,
                 'check:available': status and status >= 200 and status < 400,
                 'check:date': check_date,
             }
 
+            for header in [
+                'content-type', 'content-length', 'content-md5', 'charset'
+            ]:
+                result[f"metadata:{header}"] = response.get(header)
```

Comment:
Not sure of the prefix

Reply:
Maybe `check-metadata:` since we already use `check:` for croquemort related stuff.

Reply:
or `check-headers`